open-craft/opencraft
# Add Ansible playbook to deploy the instance manager.

**Testing instructions**

1. Bring up a PostgreSQL test server, and possibly MongoDB and MySQL test servers as well.
2. Bring up a new OpenStack server using the unmodified xenial image I uploaded to the SBG1 region of the Instance Manager production project.  (Credentials can be found on the production instance.)  A small instance will be enough, no external volume needed for testing.  (The playbooks don't mount or format any volumes.)  Edit the security groups of the database servers to give access to the new instance.
3. Follow the instructions in the `README.md` file added in this PR.  For a test private settings file, see the Jira ticket.  You need to update the IP address of the Postgres server, and possibly add the MySQL and MongoDB servers to the configuration.
